example-adapter/src/main/java/org/niis/xrd4j/exampleadapter/data/Person.java

```diff
@@ -0,0 +1,154 @@
+/**
+ * The MIT License
+ * Copyright © 2018 Nordic Institute for Interoperability Solutions (NIIS)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.niis.xrd4j.exampleadapter.data;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.niis.xrd4j.exampleadapter.data.PersonContact.PersonContactType;
+
+/**
+ * This class implements a POJO for the "person" complexType defined
+ * in the "example.wsdl" file that's located in WEB-INF/classes folder. The name 
+ * of the WSDL file and the namespace is configured in the 
+ * WEB-INF/classes/xrd-servlet.properties file.
+ *
+ * Also provides a static method for getting our example dataset.
+ *
+ * @author Raido Kaju
+ */
+public class Person {
+    private String SSN;
```

Comment:
Since `SSN` is an instance variable and not a constant, it would be better to rename it to `ssn`.

Reply:
Changed